classroom.rb

```diff
@@ -1,14 +1,13 @@
 class Classroom
-  attr_accessor :label
-  attr_reader :student
+  attr_accessor :label, :students
```

Comment:
- [ ] Kindly make `:students` in classroom only readable, we don't want to change students in the class but only add